<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetStats</class>
 <widget class="QWidget" name="NetStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>171</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="sentLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="recvLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Latency:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lagLabel">
     <property name="text">
      <string notr="true">0</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Sent:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel">
     <property name="text">
      <string>Received:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
